cmd/edv-rest/startcmd/start.go

```diff
@@ -55,6 +56,8 @@ const (
 		" Alternatively, this can be set with the following environment variable: " + databasePrefixEnvKey
 )
 
+var logger = log.New("edv/cmd/edv-rest/main")
```

Comment:
Shouldn't use package/path in module, you can already see them in caller info. refer aries for module names.

Reply:
Actually, the caller info doesn't display the full path. It only displays `PackageName.MethodName`. But indeed the package name doesn't need to be in the logger module name, so I'll remove that.